example-mojaloop-backend/values.yaml

```diff
@@ -56,121 +43,25 @@ kafka:
     persistence:
       enabled: false
 
+    ## @param fullnameOverride String to fully override  template
+    ##
+    fullnameOverride: "zookeeper"
+
     ## @param affinity Affinity for pod assignment
     ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
     ## Note: podAffinityPreset, podAntiAffinityPreset, and  nodeAffinityPreset will be ignored when it's set
     ##
     affinity: {}
 
-    ## @param nodeSelector Node labels for pod assignment
-    ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
-    ##
-    # nodeSelector: {}
-    
-    ## Example
-    # nodeSelector:
-    #     "node-role.mojaloop.io": kafka
-
-    ## @param tolerations Tolerations for pod assignment
-    ## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
-    ##
-    # tolerations: []
-    
-    ## Example
-    # tolerations:
-    #   - key: "node-role.mojaloop.io"
-    #     operator: "Equal"
-    #     value: kafka
-    #     effect: "NoSchedule"
 
 ## Reference: https://github.com/bitnami/charts/blob/main/bitnami/mysql/values.yaml
-centralledger-mysql:
+mysql:
   enabled: true
 
-  ## MySQL Authentication parameters
+  ## @param fullnameOverride String to fully override template
   ##
-  auth:
```

Comment:
Why was this removed?

Take note that persistence should always be disabled by default, otherwise default installation fail.

Reply:
@mdebarros I think we discussed but default installs only fail if pv and pvc's were not removed from previous deployment. Still per discussions it is cleaner for default install I agree and so is returned to false.